test-framework/test-suites/integration/tests/list/test_list_attr.py

```diff
@@ -0,0 +1,37 @@
+import pytest
+import json
+
+@pytest.mark.usefixtures('revert_database')
+class TestListAttr:
+
+
+	def test_list_attr_with_shadow(self, host):
+		result = host.run('stack list attr output-format=json')
+		assert result.rc == 0
+
+		result = host.run('stack list attr shadow=True output-format=json')
+		assert result.rc == 0
+
+
+
+
+	def test_list_attr_without_shadow(self, host):
+		result = host.run('stack list attr shadow=False output-format=json')
+		assert result.rc == 0
+		
+		attr_json_file = json.loads(result.stdout)
+		assert self.validate_attr_list(attr_json_file)
+
+
+
+
+	# Checks if any attr in the attr list is of the type shadow and it's corresponding value is Null or None
```

Comment:
No need to make this a method, just put it inline in your test.  If/when it gets used a bit more we can pull it back out.